styles/style.css

```diff
@@ -0,0 +1,160 @@
+/* NAVIGATION BAR */
+nav ul {
+    display: grid;
+    grid-template-columns: 6fr 2fr 1fr;
+}
+
+nav li {
+    list-style-type: none;
+}
+
+/* TAGLINE SECTION */
+.intro {
+    height: 25vh;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-around;
+    border-bottom: 1px solid grey;
+}
+
+.intro a:link, a:visited {
+    color: gray;
+    padding: 14px 25px;
+    text-align: center;
+    text-decoration: none;
+    border: 1px solid;
+    border-radius: 8px;
+    display: inline-block;
+}
+
+.intro sub {
+    margin-bottom: 5vh;
+}
+
+/* QUOTES */
+.testimonial {
+    margin-top: 10vh;
+    height: 25vh;
+    text-align: left;
+    border-bottom: 1px solid grey;
+}
+
+.testimonial p {
+    text-align: right;
+    margin-top: 10vh;
+    margin-right: 25vh;
+    
+}
+
+/* WHY SECTION */
+.selling-points {
```

Comment:
great naming of classes! "selling-points" is such a more descriptive name than "why".